src/main/java/seedu/address/model/person/MeetingTime.java

```diff
@@ -28,9 +29,14 @@ public class MeetingTime {
     public MeetingTime(String meetingTimeString) {
         requireNonNull(meetingTimeString);
         checkArgument(isValidMeetingTime(meetingTimeString), MESSAGE_CONSTRAINTS);
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy-HH:mm");
```

Comment:
is there a need to make it caps and uncaps within the pattern? like hour is HH but minute is mm

Reply:
No real purpose to different capitalisation, will just standardise all capitalised if everyone is okay with it